nexuse2e-webapp/src/main/frontend/src/app/date-range-filter/date-range-filter.component.ts

```diff
@@ -18,8 +18,14 @@ export class DateRangeFilterComponent {
 
   ngOnChanges(): void {
     if (this.selectedValue) {
-      this.startDate = this.selectedValue.startDate;
-      this.endDate = this.selectedValue.endDate;
+      if (typeof this.selectedValue === "string") {
+        const parsed = JSON.parse(this.selectedValue);
+        this.startDate = new Date(parsed.startDate);
```

Comment:
Is it worth checking if `parsed.startDate` is set?

Reply:
Thanks for the hint, this part was obsolete after some changes before.